qutebrowser/config/qtargs.py

```diff
@@ -259,3 +259,10 @@ def init_envvars() -> None:
                    if qtutils.version_check('5.14', compiled=False)
                    else 'QT_AUTO_SCREEN_SCALE_FACTOR')
         os.environ[env_var] = '1'
+
+    for var in config.val.qt.environ:
+        val = config.val.qt.environ[var]
+        if val == 'None':
+            os.environ[var] = ''
```

Comment:
Deleting an environment variable and setting it to an empty string are two different things - depending on the application reading it, they might do something different.

You can delete/unset the variable using [the `del` keyword](https://docs.python.org/3/tutorial/datastructures.html#the-del-statement) (the doc talks about lists, but the same applies to dicts as well):

```suggestion
            del os.environ[var]
```

You'll likely also need to adjust the test accordingly, or maybe even separate the test into two.